src/wallet/rpcwallet.cpp

```diff
@@ -2639,6 +2660,76 @@ static UniValue loadwallet(const JSONRPCRequest& request)
     return obj;
 }
 
+static UniValue setwalletflag(const JSONRPCRequest& request)
+{
+    std::shared_ptr<CWallet> const wallet = GetWalletForJSONRPCRequest(request);
+    CWallet* const pwallet = wallet.get();
+
+    if (!EnsureWalletIsAvailable(pwallet, request.fHelp)) {
+        return NullUniValue;
+    }
+
+    if (request.fHelp || request.params.size() < 1 || request.params.size() > 2) {
```

Comment:
nit: Use `RPCHelpMan help{...}`. See https://github.com/bitcoin/bitcoin/blob/master/src/rpc/rawtransaction.cpp#L74 for an example.

Reply:
Will address if rebase becomes necessary, otherwise will try to preserve ACKs and do a quick follow-up post-merge.

Reply:
I can do this but it would mean some unnecessary calculations when users are not asking for help:

https://github.com/bitcoin/bitcoin/blob/44d81723236114f9370f386f3b3310477a6dde43/src/wallet/rpcwallet.cpp#L2671-L2674

Still worth it?